src/Development/IDE/Core/Completions.hs

```diff
@@ -361,6 +361,7 @@ getCompletions ideOpts CC { allModNamesAsNS, unqualCompls, qualCompls, importabl
       filtImportCompls = filtListWith (mkImportCompl enteredQual) importableModules
       filtPragmaCompls = filtListWithSnippet mkPragmaCompl validPragmas
       filtOptsCompls   = filtListWith mkExtCompl
+      filtKeywrdCompls = if T.null prefixModule then filtListWith mkExtCompl keywords else []
```

Comment:
```suggestion
      filtKeywordCompls = if T.null prefixModule then filtListWith mkExtCompl keywords else []
```
Looks like a typo? Or is that intentional to match length of the other compls?

Reply:
The latter 🙄

Reply:
Just don’t align things. It only makes diffs noisy and introducing typo-like things to keep alignment is confusing.

Reply:
👍 I've commited the suggestion

Reply:
I am sorry, but the suggestion is probably not enough, arent you using the function in another location?